.devcontainer/devcontainer.json

```
@@ -0,0 +1,23 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+  "name": "Node.js & TypeScript",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:1-22-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/github-cli:1": {
+      "installDirectlyFromGitHubRelease": true,
+      "version": "latest"
+    },
+    "ghcr.io/devcontainers-extra/features/pnpm:2": {
+      "version": "latest"
+    }
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": ["aaron-bond.better-comments", ""]
+    }
+  },
+  "postCreateCommand": "pnpm install && pnpm postinstall",
```

Isn't `postinstall` supposed to run automatically after you run `install`?

---

🤷

---

```suggestion
  "postCreateCommand": "pnpm install",
```
https://pnpm.io/cli/run#enableprepostscripts
(Same as `npm`)